src/QMCDrivers/VMC/VMCUpdateReny.cpp

```diff
@@ -0,0 +1,569 @@
+//////////////////////////////////////////////////////////////////////////////////////
```

Comment:
OK. Unless someone claims ownership and produces tests and inputs, this code is likely to be removed shortly.